examples/example-vite-svelte-recs/src/dojo/typescript/models.gen.ts

```
@@ -0,0 +1,193 @@
+  pending_word: BigInt;
+  pending_word_len: Number;
+}
+export const ByteArrayDefinition = {
+  data: RecsType.StringArray,
+  pending_word: RecsType.BigInt,
+  pending_word_len: RecsType.Number,
+};
+
+// Type definition for `dojo::model::layout::FieldLayout` struct
+export interface FieldLayout {
+  selector: BigInt;
```

Biome

[error] Don't use 'BigInt' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'bigint' instead


(lint/complexity/noBannedTypes)